features/networking/ptp_operator.feature

```diff
@@ -11,3 +11,101 @@ Feature: PTP related scenarios
       | app=linuxptp-daemon |
     And status becomes :running of exactly 1 pods labeled:
       | name=ptp-operator |
+
+  # @author huirwang@redhat.com
+  # @case_id OCP-25738
+  @admin
+  Scenario: Linuxptp run as slave can sync to master
+    Given the ptp operator is running well
```

Comment:
Thanks Huiran. So are we leaving ptp operator running on shared cluster after such tests are done? Won't we need a tear_down procedure under same def to leave cluster in its originality?

Reply:
Also your ```ptp operator is running well``` step has initiated the flow as admin user. Do we need further steps in all scenario to be executed as admin?

Reply:
Thanks Anurag, the ptp cases only can be run in real baremetal env.  We don't want to deploy ptp operator every time in the  case so just leave the operator there for the following cases to be executed on. It is okay to leave it there.

Reply:
For the second question, thanks point out, fixed it.